src/lib/tls/tls_policy.h

```diff
@@ -420,12 +420,12 @@ class BOTAN_PUBLIC_API(2,0) BSI_TR_02102_2 : public Policy
    public:
       std::vector<std::string> allowed_ciphers() const override
          {
-         return std::vector<std::string>({"AES-256/GCM", "AES-128/GCM", "AES-256", "AES-128" });
+         return std::vector<std::string>({"AES-256/GCM", "AES-128/GCM", "AES-256", "AES-128", "AES-256/CCM", "AES-128/CCM"});
```

Comment:
Are you sure you want to prioritize CBC over CCM?

Reply:
Hmm, not really. Changed the order in 4fca820cc70826451881dbe5410b17efd9ceee96.